fe/src/main/java/org/apache/doris/analysis/SelectStmt.java

```diff
@@ -349,6 +350,19 @@ public void analyze(Analyzer analyzer) throws AnalysisException, UserException {
             }
         }
         if (groupByClause != null && groupByClause.isGroupByExtension()) {
+            for (SelectListItem item : selectList.getItems()) {
```

Comment:
Please Add a UT. Currently we have a FE UT frame, which could handle SQL analyze bug. you could refer to `DemoTest`.